src/contracts/core/DelegationManagerStorage.sol

```diff
@@ -26,33 +28,43 @@ abstract contract DelegationManagerStorage is IDelegationManager {
         "DelegationApproval(address delegationApprover,address staker,address operator,bytes32 salt,uint256 expiry)"
     );
 
+    /// @notice Minimum withdrawal delay in seconds until all queued withdrawals can be completed.
+    uint32 public immutable MIN_WITHDRAWAL_DELAY;
+
     /**
      * @notice Original EIP-712 Domain separator for this contract.
      * @dev The domain separator may change in the event of a fork that modifies the ChainID.
      * Use the getter function `domainSeparator` to get the current domain separator for this contract.
      */
     bytes32 internal _DOMAIN_SEPARATOR;
 
+    /// @notice The AVSDirectory contract for EigenLayer
+    IAVSDirectory public immutable avsDirectory;
+
+    // TODO: Switch these to ShareManagers, but this breaks a lot of tests
+
```

Comment:
Let's move `_DOMAIN_SEPARATOR` below the immutables, so we keep all those in the same place.